<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 240, 246)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Что вам известно в прямоугольнике?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">);</string>
        </property>
        <property name="text">
         <string>Две стороны</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="side_a">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>10000000000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>b</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="side_b">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>100000000000000004764729344.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="side_run">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(156, 185, 209)</string>
          </property>
          <property name="text">
           <string>Рассчитать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="side_ans"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Диагональ и одну сторону</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="side_diagonal_a">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>10000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>d</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="side_diagonal_d">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>1000000000000000000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="side_diagonal_run">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(156, 185, 209)</string>
          </property>
          <property name="text">
           <string>Рассчитать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="side_diagonal_ans"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Диагональ и угол между ними</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>d</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="diagonal_d">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>999999999.990000009536743</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>α</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="diagonal_aa">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="diagonal_run">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(156, 185, 209)</string>
          </property>
          <property name="text">
           <string>Рассчитать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="diagonal_ans"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Сторона и радиус описанной окружности</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="radius_and_side_a">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>9999999999999.990234375000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="radius_and_side_R">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>99999999999.990005493164063</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="radius_and_side_run">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(156, 185, 209)</string>
          </property>
          <property name="text">
           <string>Рассчитать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="radius_and_side_ans"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Сторона и диаметр описанной окружности</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="diametr_and_side_a">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="maximum">
           <double>9999999999.989999771118164</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="diametr_and_side_D">
          <property name="styleSheet">
           <string>0</string>
          </property>
          <property name="maximum">
           <double>999999999999.989990234375000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="diametr_and_side_run">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(156, 185, 209)</string>
          </property>
          <property name="text">
           <string>Рассчитать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="diametr_and_side_"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="rectangle_1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rectangle_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="back">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 122, 172);</string>
      </property>
      <property name="text">
       <string>&lt;--</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="question">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 122, 172);</string>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
